src/Helper/HeadLink.php

```diff
@@ -383,8 +387,10 @@ public function createData(array $attributes)
     /**
      * Create item for stylesheet link item
      *
+     * @deprecated This method is unused and will be removed in version 3.0 of this component
+     *
      * @param  array $args
-     * @return stdClass|false Returns false if stylesheet is a duplicate
+     * @return object|false Returns false if stylesheet is a duplicate
```

Comment:
IMO we should hold this back a bit - this change is potentially breaking

Reply:
That's a deprecated, undocumented and internally unused method and it still actually returns a `stdClass`. For HeadLink specifically, the internal type is just `(object) array<string, mixed>`, in other implementations, documented return types have been changed to Psalm object shapes. I can't see any practical way that BC is broken here.

Returning `object` here just satisfies the template for `TValue`.

Can you explain how this might be a BC break?

Reply:
The expectation downstream is `stdClass`,  not `object`, heh :D